experimental/regression_suite/shark-test-suite-models/sd3/test_clip.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+from ireers import *
+import os
+
+repo_root = os.getenv("TEST_SUITE_REPO_ROOT")
+current_dir = repo_root + "/iree_special_models/sd3/prompt-encoder"
```

Comment:
I'm still on the fence about how best to architect this, but my gut says this is still inverted and awkward to use.

The `experimental/regression_suite` code (which is stale / unmaintained) uses code like
```python
llama2_7b_f16qi4_stripped_source = fetch_source_fixture(
    "https://sharktank.blob.core.windows.net/sharktank/llama_regression/09152023/llama2_7b_int4_stripped.mlir",
    group="llama2_7b_f16qi4_stripped",
)
```
that fetches as part of the test and doesn't rely on a git repo existing somewhere on disk.

similarly, some new sharktank tests (https://github.com/nod-ai/sharktank/blob/main/sharktank/integration/models/punet/integration_test.py) do this:
```python
@pytest.fixture(scope="module")
def sdxl_fp16_base_files():
    from huggingface_hub import hf_hub_download

    REPO_ID = "stabilityai/stable-diffusion-xl-base-1.0"
    REVISION = "76d28af79639c28a79fa5c6c6468febd3490a37e"

    def download(filename):
        return hf_hub_download(
            repo_id=REPO_ID, subfolder="unet", filename=filename, revision=REVISION
        )

    return {
        "config.json": download("config.json"),
        "params.safetensors": download("diffusion_pytorch_model.fp16.safetensors"),
    }
```

(Stella wrote both of those tests and it's a style I like but am still getting used to)

I'd prefer that style of test file fetching here over the `download_remote_files.py` script, so developers can run `pytest` directly without any other repo fetching: https://github.com/iree-org/iree/tree/main/experimental/regression_suite#quick-start

The main reasons to put files outside of the core IREE repo are:
* Keeping the core repo focused (lower commit volume, lower complexity, more shared between backends / contributors)
* Sharing files across multiple projects (e.g. out of tree plugins like https://github.com/nod-ai/iree-amd-aie)